interface/reports/appointments_report.php

```diff
@@ -345,7 +346,7 @@ function refreshme() {
     $showDate = ($from_date != $to_date) || (!$to_date);
     ?>
 <div id="report_results">
-<table class='table'>
+<table class='table' id="export">
```

Comment:
The table has to be tagged with an id. It can be anything. But it must also be changed in the JS file.  So, for now, all tables that use this JS would have to be labeled with the id of export.

Reply:
Class or even data attribute rather than id would be better marker - something like `class='table table-csvexport'` and allow java code to handle multiple exports or combine all exportable tables into a single file.